test/test_models.py

```diff
@@ -0,0 +1,158 @@
+from freezegun import freeze_time
+
+from src.models import (
+    ActivityTypes,
+    SuperUserRoles,
+    create_user,
+    create_superuser,
+    create_activity,
+)
+import pytest
+from datetime import datetime
+from pydantic import ValidationError
+
+
+@pytest.fixture
```

Comment:
you can define the scope of the fixture if you're calling it in several tests. That way they're created only once per scope, and not every time they're called.

Reply:
Added scope :)